### ISSUE TITLE
v41.0 NBMiner does not recognize GA103 3060ti's as LHR

### ISSUE THREAD

**Issue by what1ncewas** [NebuTech/NBMiner#830]
v41.0 NBMiner **does not** recognize GA103 3060ti's as LHR

v41.0 NBMiner **does** recognize GA104 3060ti's as LHR

Driver 512.15 for WIndows

Zotac cards - GA103 in GPU-Z - 33 mh/s
MSI cards - GA104 in GPU-Z - 62 mh/s

**Comment by heliosgnosis** [comment]
use no overclocks or power control yet to see if she detects them???

**Comment by what1ncewas** [comment]
Yes, I tried it with and without OC as well as re-install drivers. The only thing I did not do was disconnect
the gpus from the miner itself. 

Other applications detect them as LHR

**Comment by newbee008** [comment]
> 

My 2 GA 104 3060ti only at 49MHz. The reason is that core clock is too low and i can't lock it with NBminer and Afterburner when the NBminer is running. Anyone has a solution?

[21:35:28] INFO - |ID|Device|Hashrate|Accept|Reject|Inv|Powr|CTmp|MTmp|Fan|CClk|GMClk|MUtl|Eff/Watt|
[21:35:28] INFO - | 0| 3060| 48.23 M| 487| 0| 0| 118| 64| | 69|1132| 8400| 100| 408.7 K|
[21:35:28] INFO - | 1|3060ti| 51.97 M| 518| 0| 0| 130| 54| | 57| 690| 6920| 100| 399.8 K|
[21:35:28] INFO - | 2|3060ti| 49.78 M| 510| 0| 0| 130| 50| | 49| 615| 6790| 100| 382.9 K|
[21:35:28] INFO - |------------------+------+------+---+----+--------------------------------------|
[21:35:28] INFO - | Total: 150.0 M| 1515| 0| 0| 378| Uptime: 0D 12:10:01 CPU: 1% |
